client/app/components/queries/editor-components/index.js

```diff
@@ -0,0 +1,10 @@
+import SchemaBrowser from "@/components/queries/SchemaBrowser";
+import QueryEditor from "@/components/queries/QueryEditor";
+
+import { registerEditorComponent, useEditorComponents } from "./editorComponents";
+
+// default query editor components
+registerEditorComponent("SchemaBrowser", SchemaBrowser);
+registerEditorComponent("QueryEditor", QueryEditor);
```

Comment:
The names should be constants defined in `editorComponents`.

Reply:
Good one, done 👍